package.json

```diff
@@ -51,6 +51,7 @@
   "ignore": {},
   "devDependencies": {
     "babel-register": "^6.26.0",
+    "chai": "^4.2.0",
```

Comment:
I'm reluctant to bring in another assertion lib when we've already got `jest` which provides `expect`. I'm guessing this was the path of least resistance and that grid shell used this. Do you think converting to `expect` is worth the effort?

Reply:
Actually, this is trivial. Knocking it out now.

Reply:
Oops didn't realize we had jest in there already. Happy to convert it if you haven't already.

Reply:
no worries! almost done.